packages/swingset-runner/bin/mainLauncher

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S node -r esm
+
+import process from 'process';
+import path from 'path';
+
+/**
+ * Simple boilerplate program providing linkage to launch an application written using modules within the
+ * as yet not-entirely-ESM-supporting version of NodeJS.
+ */
+const { main } = require(`../src/${path.basename(process.argv[1])}.js`);
```

Comment:
Hrm, you know, this is a weird mix of ESM and `require` that I don't think I can support. Plus it introduces a surprising `mainLauncher` into the user's $PATH which can't do anything on its own (and will cause all sorts of confusion if run directly).

And it looks like the only thing being factored out is the shbang line. How about instead `bin/graphTime` does:

```js
#!/usr/bin/env -S node -r esm

import { main } from '../src/dataGraph.js';
main();
```

and we just copy those three lines into each of the three tools?

Reply:
I don't follow.  The old graphTime looked like this:
```
#!/usr/bin/env -S node -r esm

import { main } from '../src/graphTime.js';
main();
```
with corresponding versions for each graphXXX utility.  The idea here was to avoid having to write a separate piece of boilerplate for each, since they were all the same modulo the thing they invoked.